doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -246,6 +246,10 @@ nRF Desktop
     * Disabled Bluetooth long workqueue (:kconfig:option:`CONFIG_BT_LONG_WQ`).
     * Limited the number of key slots in the PSA Crypto core to 10 (:kconfig:option:`CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT`).
 
+  * Disabled the :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ` Kconfig option in the IPC radio image configuration for the nRF54H20 SoC.
+    The :ref:`nrf_desktop_dvfs` running on the application core selects the clock control and enables an automatic low Global Domain (GD) frequency request (:kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ`).
```

Comment:
That is not correct, or complete picture. To make the GD frequency low all local domains has to setup this frequency. The CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ is enabled because the nrf2 clock control is enabled. I don't know what enables it in application core. In radio core the nrf2 clock control is enabled by MPSL.

Reply:
I emphasized what enables it on the app core here:
```
The :ref:`nrf_desktop_dvfs` running on the application core selects the clock control and enables an automatic low Global Domain (GD) frequency request (:kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ`).
```

Reply:
Introduced a bit more details